spec/lib/events/s3_exporter_spec.rb

```diff
@@ -23,9 +23,10 @@
   end
 
   def build_csv(expected_events)
-    columns = "id,action,payload,user_uid,created_at,updated_at,request_id,content_id\n"
+    columns = "id,action,user_uid,created_at,updated_at,request_id,content_id,payload\n"
     expected_events.inject(columns) do |memo, event|
-      memo << CSV.generate_line([event.id, event.action, event.payload.to_json, event.user_uid, event.created_at, event.updated_at, event.request_id, event.content_id])
+      event.reload
+      memo << CSV.generate_line([event.id, event.action, event.user_uid, event.created_at, event.updated_at, event.request_id, event.content_id, event.payload.to_json])
```

Comment:
Will these changes cause issues when restoring the CSV files?

Reply:
My understanding is that it won't cause any issues because the importer parses the fields in any order they come in: https://github.com/alphagov/publishing-api/blob/4028a4e690d1ac5b1ca0b9fce268b2f0bd892d03/lib/events/s3_importer.rb#L37-L43

Reply:
Good good :+1: I think it's rare that we import data from S3 anyway.